app.py

```diff
@@ -14,196 +18,12 @@
-    :return: Dict with Node attributes
-    """
-    node = {
-        "id": id,
-        "shape": shape,
-        "size": size,
-        "label": label
-    }
-    return node
-
-
-def create_edge(from_, to, label=None):
-    """Auxiliary function that creates a Edge in Vis.js format
-    :return: Dict with Edge attributes
-    """
-    edge = {
-        "from": from_,
-        "to": to
-    }
-    if label is not None:
-        edge["label"] = label
-    return edge
-
-@app.route("/send-command", methods=['POST'])
-def send_command():
-    """Send Command to Agent and returns hydrus response
-    :param: All parameters enabled by the Agent.
-    :param: Please check Agent.py in Agent's main repository.
-    :return: hydrus response to the request
-    """
-    global agent
-    if agent is None:
-        return "No agent connected."
-    body = request.get_data()
-    body = body.decode('utf8')
-    body = json.loads(body)
-    if "method" not in body:
-        return "Request must have a method."
-    if body['method'] == "get":
-        # Get optional parameters
-        filters = body.get('filters', {})
-        cached_limit = body.get('cached_limit', sys.maxsize)
-        if 'url' in body:
-            return json.dumps(agent.get(url=body['url'], filters=filters,
-                              cached_limit=cached_limit))
-        elif 'resource_type' in body:
-            return json.dumps(agent.get(resource_type=body['resource_type'],
-                              filters=filters, cached_limit=cached_limit))
-        else:
-            return "Must contain url or the resource type"
-    elif body['method'] == "put":
-        if "url" in body:
-            url = body['url']
-        else:
-            return "Put request must contain a url"
-        if "new_object" in body:
-            new_object = body["new_object"]
-        else:
-            return "Put request must contain the new_object."
-        return json.dumps(agent.put(url, new_object))
-    elif body['method'] == "post":
-        if "url" in body:
-            url = body['url']
-        else:
-            return "Post request must contain a url"
-        if "updated_object" in body:
-            updated_object = body["updated_object"]
-        else:
-            return "Put request must contain the updated_object."
-        return json.dumps(agent.post(url, updated_object))
-    elif body['method'] == "delete":
-        if "url" in body:
-            url = body['url']
-        else:
-            return "Delete request must contain a url"
-        return json.dumps(agent.delete(url))
-    else:
-        return "Method not supported."
+app.add_url_rule('/<path:path>','serve',serve)
+app.add_url_rule('/start-agent','start-agent', start_agent)
+app.add_url_rule('/hydra-doc','hydra-doc', hydra_doc)
+app.add_url_rule('/apidoc-graph','apidoc-graph', apidoc_graph)
+app.add_url_rule('/send-command','send-command', send_command)
+app.add_url_rule('/','default',default)
```

Comment:
You should try to follow PEP8 convention here. Try to give a space after every comma in the function parameter list. 
`app.add_url_rule('/<path:path>','serve',serve)`  -> `app.add_url_rule('/<path:path>', 'serve', serve)`